sagemaker/launch/launch_train.py

```diff
@@ -109,7 +109,7 @@ def run_job(input_args, image, unknowargs):
         py_version="py3",
         base_job_name=prefix,
         hyperparameters=params,
-        sagemaker_session=sess,
+        # sagemaker_session=sess,
```

Comment:
Why we comment here?

Reply:
Same question.

Reply:
This allows us to launch and test SageMaker jobs using SageMaker [local mode](https://sagemaker.readthedocs.io/en/stable/overview.html#local-mode) by simply setting the instance-type to `local`. I used this functionality to quickly test my changes locally before launching actual SM jobs.